Library/Homebrew/dev-cmd/pr-upload.rb

```diff
@@ -101,15 +101,14 @@ def pr_upload
     bottle_args += json_files
 
     if args.dry_run?
-      service =
-        if internet_archive?(bottles_hash)
+      service = if github_packages?(bottles_hash)
+          return
```

Comment:
This change gets the sense of the test backwards. It's supposed to return early _unless_ uploading to GitHub Packages. Now it's returning early _if_ uploading to GitHub Packages.

Reply:
@sjackman yes 🤦🏻